apps/prairielearn/src/models/course-instance-usages.sql

```diff
@@ -74,6 +74,9 @@ FROM
   LEFT JOIN course_instances AS ci ON (ci.course_id = a.course_instance_id)
 WHERE
   gj.id = $grading_job_id
+  -- Avoid inserting anything if we'd compute a NULL duration.
+  AND gj.grading_received_at IS NOT NULL
+  AND gj.grading_finished_at IS NOT NULL
```

Comment:
This takes roughly the same approach as the backfill migration, where we avoid inserting anything at all instead of just inserting a `0` duration:

https://github.com/PrairieLearn/PrairieLearn/blob/56b9b21d705095b42dde7c188bce2cba2cb893b5/apps/prairielearn/src/batched-migrations/20250214171608_course_instance_usages__external_grading_jobs__backfill.sql#L59

I noticed that the backfill doesn't handle a `NULL` value for `grading_received_at`. I opted to handle that here to be extra safe. Maybe we want to do the same in the backfill?

Reply:
Good call: https://github.com/PrairieLearn/PrairieLearn/pull/11448